src/core/qzmq/src/qzmqsocket.cpp

```diff
@@ -612,6 +615,13 @@ class Socket::Private : public QObject
 		}
 	}
 
+	void update_timeout()
```

Comment:
this slot is still being used in the old QT style way. should we move it out of `slot` section?

Reply:
it is not in the slot section afaict